lib/rucio/common/test_rucio_server.py

```diff
@@ -14,11 +14,9 @@
 # limitations under the License.
 #
 # Authors:
-# - Joaquín Bogado <jbogado@linti.unlp.edu.ar>, 2014-2018
```

Comment:
Think we can keep the old authors?

Reply:
nvm, it's you 3 folks in the git blame ;) Not sure where the other list got here from.

Reply:
I also have no idea...